packages/vue/src/components/atoms/SfBreadcrumbs/SfBreadcrumbs.md

```diff
@@ -0,0 +1,46 @@
+# component-description
+Component which renders a breadcrumb with router links for indicating the level of navigation the user is currently in.
+
+# common-usage
+<br>
+<SfDocsBreadcrumbs />
+
+```html
+<template>
+  <SfBreadcrumbs :breadcrumbs="breadcrumbs" />
+</template>
+
+<script>
+import { SfBreadcrumbs } from "@storefront-ui/vue";
+
+export default {
+  components: {
+    SfBreadcrumbs
+  },
+  data() {
```

Comment:
This is exactly what i though I will suggest when I saw that you took breadcrumbs, then I saw you already did it :D